build/configs/artik05x/artik05x_download.sh

```diff
@@ -67,10 +69,28 @@ Options:
 EOF
 }
 
+prepare_resource()
+{
+    if [ -d "${RESOURCE_DIR_PATH}" ]; then
+        echo "Packing resources into romfs.img ..."
```

Comment:
romfs.img is not created in this snippet, only content is being copied.. So do we need this message?